src/main/java/org/osiam/resources/controller/ResourceController.java

```diff
@@ -64,4 +74,29 @@
         result.add("id");
         return result;
     }
+
+    protected MappingJacksonValue buildWrapper(T resource, String attributes) {
```

Comment:
Because one builds the wrapper for `SCIMSearchResult<T>` and the other for `T`. I am going to add the third method.